tools/clang_tidy/lib/clang_tidy.dart

```diff
@@ -171,23 +189,82 @@ class ClangTidy {
     return repo.changedFiles;
   }
 
+  Iterable<T> _takeShard<T>(Iterable<T> values, int id, int shardCount) sync*{
```

Comment:
I guess dart format isn't running on this package?
```suggestion
  Iterable<T> _takeShard<T>(Iterable<T> values, int id, int shardCount) sync* {
```

Reply:
As with the framework repo, dartfmt isn't run in the engine repo.